src/main.rs

```diff
@@ -136,6 +145,7 @@ fn _main() -> Result<(), anyhow::Error> {
             workspace,
             offline,
             no_std,
+            bin: _bin,
```

Comment:
Shouldn't we pass this over in order to invoke `cargo build` with the `--bin` flag?

Reply:
Added in 438f86a! Thank you for the review!